src/main/java/com/hubspot/jinjava/lib/tag/ForTag.java

```diff
@@ -129,6 +130,7 @@ public String interpret(TagNode tagNode, JinjavaInterpreter interpreter) {
       LengthLimitingStringBuilder buff = new LengthLimitingStringBuilder(interpreter.getConfig().getMaxOutputSize());
       while (loop.hasNext()) {
         Object val = loop.next();
+        val = interpreter.resolveProperty(val, Collections.emptyList());
```

Comment:
I see what you're trying to do here, but it's weird to call `resolveProperty` with an empty list of properties to resolve. Perhaps it would make more sense just to add a delegate method in `JinjavaInterpreter` for `wrap`.

Reply:
👍 updated